cli/plasmo/src/commands/build.ts

```diff
@@ -1,25 +1,15 @@
 import { paramCase } from "change-case"
 import { createWriteStream } from "fs"
-import { emptyDir, ensureDir } from "fs-extra"
+import { emptyDir } from "fs-extra"
 import { resolve } from "path"
 
-import {
-  getFlag,
-  getNonFlagArgvs,
-  hasFlag,
-  iLog,
-  sLog,
-  vLog
-} from "@plasmo/utils"
+import { getFlag, getNonFlagArgvs, hasFlag, iLog, sLog } from "@plasmo/utils"
```

Comment:
Does your prettier have something like this in it? 

```ts
this.config.eachImportStatementOnNewLine = DAY_OF_WEEK % 3
```

🤣

Reply:
it got shortened tho :d